src/common/btree.c

```diff
@@ -15,6 +15,7 @@
 
 #include <daos_errno.h>
 #include <daos/btree.h>
+#include <daos/dlck.h>
```

Comment:
i don't think we should add any sort of dependencies to dlck from the common code here, even for just #defines. If we need some infrastructure to just print some information, it should be self-contained into the common library IMO.